.gitignore

```diff
@@ -8,3 +8,5 @@ storybook-static
 .#*
 packages/**/yarn.lock
 packages/**/package-lock.json
+.coveralls.yml
```

Comment:
did you mean to commit and add to .gitignore?

Reply:
Well.. probably not, but there's no reason to modify it after so we could just leave it as is.